sql/updates/reverse-dev.sql

```diff
@@ -75,3 +75,32 @@ BEGIN
     );
 END
 $$ LANGUAGE 'plpgsql';
+
+--undo compression feature for caggs
+--check that all continuous aggregates have compression disabled
+--this check is sufficient as we cannot have compressed chunks, if
+-- compression is disabled
+DO $$
+DECLARE
+  cagg_name NAME;
+  cnt INTEGER := 0;
+BEGIN
+    FOR cagg_name IN
+        SELECT view_name, 
+               materialization_hypertable_schema ,
+               materialization_hypertable_name   
+        FROM timescaledb_information.continuous_aggregates
+        WHERE compression_enabled is TRUE
+    LOOP
+        RAISE NOTICE 'Compression is enabled for continuous aggregate: %', cagg_name;
+        cnt := cnt + 1; 
+    END LOOP;
+    IF cnt > 0 THEN
+       RAISE EXCEPTION 'Cannot downgrade as compression is enabled for continuous aggregates' 
```

Comment:
The downgrade test runs as a cron job as a github action. Its not clear to me what versions we are testing against and if a downgrade test can be added for this feature before 2.6. is released. I'll defer this until Mats is back - he added the downgrade testing script. I don't want to risk breaking master.

Reply:
The downgrade tests are quite simplistic at this point. They basically run the following sequence:

1. Create the lower version X
2. Run the post scripts and dump them.
2. Upgrade to higher version Y
3. Immediately downgrade to version X again.
4. Run the post scripts and make sure that the results match what was there before.

To run this kind of test, we need two things:

- Add the ability to run some things after the upgrade. This was not added before since we did not intend to support anything but an immediate downgrade. If you make changes, to the installation after that, we cannot guarantee that a downgrade will work, so we are not testing this.
- Add the ability to check for the failure case.
Both of these are possible, but it is quite a lot of work in the current state.